test/vendor/tracekit.test.js

```diff
@@ -134,6 +134,39 @@ describe('TraceKit', function(){
             }
         });
 
+        describe('when no 5th argument (error object)', function () {
+            it('should seperate name, message for default error types (e.g. ReferenceError)', function (done) {
+                subscriptionHandler = function (stackInfo, extra) {
+                    assert.equal(stackInfo.name, 'ReferenceError');
+                    assert.equal(stackInfo.message, 'foo is undefined');
+                    done();
+                };
+                TraceKit.report.subscribe(subscriptionHandler);
+                window.onerror('Uncaught ReferenceError: foo is undefined', 'http://example.com', 1337);
```

Comment:
Do you think it's worth restricting the regex down then to `/^(?:Uncaught )?` instead of being liberal with `.*` ?